lib/oli_web/templates/layout/delivery.html.eex

```diff
@@ -31,7 +31,7 @@
 
     <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
     <script>
-    window.MathJax = {
+    window.MathJax = window.MathJax || {
```

Comment:
Tell MathJax not to re-initialize itself (which happens if swapping from authoring to delivery in preview mode)